ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -482,15 +482,16 @@ VAV_RD_CO2C:
   - RD
   - CO2C
 
-VAV_SDM_CO2C_SRC:
+VAV_CO2C_SRC:
   description: "CO2 controlled VAV, with damper, hi/low flowrate sensor and setpoint and schedule run"
   is_canonical: true
   implements:
   - VAV
-  - SDM
   - CO2C
   - SRC
   uses:
   - low_limit_supply_air_flowrate_setpoint
   - high_limit_supply_air_flowrate_setpoint
   - supply_air_flowrate_sensor 
+  - supply_air_damper_command
```

Comment:
Why does this have a two position damper and a percentage sensor? Is that intended?

Also, where is the supply air flowrate setpoint? I see the high/low limits, but I do not see the actual setpoint. 

If this is intended to model new functionality (i.e. there is no existing abstract type that covers this) we should create it; if its unique and we don't wish to give it an abstract type, it should be non-canonical (`is_canonical: false`). But please advise on the questions above first, because I do not understand the device's operation..

Reply:
These are the points list I have sir. I contacted our engineers for the device details/operation (dampers) and they have yet to get back to me on this. Can I ask your help and infer from this list? the only setpoint I see is the CO2, is it possible the CO2 has relationship with the dampers? 

```
VAV
--
CO2 Setpoint
Minimum Air Flow Rate Setpoint
Maximum Air Flow Rate Setpoint
Air Flow Rate Present Value
VAV Damper Control
VAV Damper Feedback
Time Schedule
```

Thanks sir